### PR TITLE
Js park file created

### PR DESCRIPTION
**Pull request by BismarTrujillo**
Javascript file for park created and linked

### CODE REVIEW

File: parks.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+
+
+
+
+
+
+
+<script src="scripts/parks_page.js" ></script>
```
Comment (mattchristenson) [inline parks.html:9]: This is really great code, good idea!

Comment (mattchristenson) [reply parks.html:9]: Why did you choose to name this "parks_page" instead of "parks"?

Comment (BismarTrujillo) [reply parks.html:9]: I like parks_page!

**Review by mattchristenson — APPROVED** [review]
Great job!

**Pull request merged**